ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -158,15 +158,11 @@
    "If SubscriberId is provided, use it. Else, get it from the token.
    If subscriber EmailAddress is provided, use it. Else, use the value supplied by EDL for the user."
   [context metadata]
-  (let [{subscriber :SubscriberId
-         email :EmailAddress} metadata
+  (let [{subscriber :SubscriberId} metadata
         subscriber (if-not subscriber
                      (api-core/get-user-id-from-token context)
```

Comment:
In accordance with Leo's comments on my last PR - no reason not to simplify this `if-not` to an `if`

Reply:
This is also where the merge conflict exists - just use your code, it supersedes mine